internal/pkg/earpointer/analysis.go

```diff
@@ -354,13 +403,25 @@ func (vis *visitor) visitCall(call *ssa.CallCommon, instr ssa.Instruction) {
 	}
 
 	// Collect unification constraints
-	// Here only static calls are supported.
-	// A future extension is to use a more advanced call graph.
+	// Here only static calls are supported;
+	// an extension is to use a more advanced call graph.
 	fn := call.StaticCallee()
 	if fn == nil {
 		return
 	}
 	paramCstrs, retCstrs := vis.collectCalleeConstraints(call, fn, instr)
+	state := vis.state
+	contextSensitiveUnify := func(callerV ssa.Value, calleeV ssa.Value) {
+		for _, c := range vis.getContexts(callerV) {
+			callerCxt := append(*c, instr)
+			for _, calleeCxt := range vis.getContexts(calleeV) {
+				if contextKEqual(callerCxt, *calleeCxt) {
+					state.Unify(MakeReference(c, callerV),
```

Comment:
Either 1 line or add reference variables:

callerFromC := MakeReference(c, callerV)
calleeFromCallee := MakeReference(calleeCxt, calleeV)
state.Unify(callerFromC, calleeFromCallee)

Reply:
I am not sure what the unification you are going for in this loop is. Can you add a comment?